### ISSUE TITLE
Parenthesis within String Literals

### ISSUE THREAD

**Issue by Yuxin45** [SAG-KeLP/kelp-full#18]
KeLP cannot ignore parenthesis within the string literal
e.g. tree representation ("str+("), it would crash because it has an open parenthesis inside the string.

**Comment by Hareem-E-Sahar** [comment]
Yes I encountered this as well and it leads to unequal parenthesis error.
